Neos.Flow/Configuration/Development/Settings.yaml

```diff
@@ -43,3 +43,29 @@ Neos:
         backendOptions:
```

Comment:
Shouldn't these old settings be removed now?

Reply:
Duh, ofc 😆 Did that now.